v3.0/reference/configuration/tidb-server/configuration.md

```diff
@@ -1,7 +1,6 @@
 ---
 title: TiDB 配置参数
 category: reference
-aliases: ['/docs-cn/op-guide/configuration/','/docs-cn/sql/server-command-option/','/docs-cn/sql/tidb-server/']
```

Comment:
add it back

Reply:
Add back in https://github.com/pingcap/docs-cn/pull/1495/commits/8ea96d204e58ed7a18ca67875b63d679ffba950e